src/Login/Login.css

```diff
@@ -0,0 +1,166 @@
+body{
```

Comment:
I made the comment here but the same applies to all .css files.

Some recommendations:
- Always add classnames to your screens
- Have a .css file for each screen (only if needed in some cases you don't have to)
- If you need to change the style of an element, add a className to that element and then select it from the css, avoid selecting by id, by element type, etc (only in some specific cases is correct to do it)
- On the .css of your screen always select first you screen className to avoid applying the style to other elements in other places. 
For example, you want to change the style of an `<h1>` with the className "title" inside your login screen with className `login-screen`. To do so:

```
.login-screen .title {
    /* change your text styles here */
}
```
- check the zeplin styling, zeplin is like a guide rather than a code to copy-paste. Sometimes it adds extra (not useful) properties.
- try to use constants for the colors (I know that on the style fix PR the is a file with all the colors, try to use them).
- for layout try using flexbox, is quite simple for most cases.

Some useful links:
- [CSS selectors](https://css-tricks.com/how-css-selectors-work/)
- [FlexBox guide](https://css-tricks.com/snippets/css/a-guide-to-flexbox/)